datadog/plugins/mongodb.sls

```diff
@@ -0,0 +1,18 @@
+{% set dd_mongo = salt.pillar.get(
+    'datadog:integrations:mongodb',
+    {
+        'passwd': 'Ch@ng3me!',
+        'password': 'N0tS3cure!'
```

Comment:
I know 'passwd' and 'password' are the names of the params for `mongodb_user.present`, but can we use something more descriptive for the var names? Perhaps `admin_password` and `datadog_password`.